pkgs/epnix/support/adsDriver/default.nix

```
@@ -0,0 +1,36 @@
+{
+  mkEpicsPackage,
+  fetchFromGitHub,
+  epnix,
+  boost,
+  epnixLib,
+  lib,
+}:
+mkEpicsPackage rec {
+  pname = "adsDriver";
+  version = "3.1.0";
+
+  src = fetchFromGitHub {
+    owner = "Cosylab";
+    repo = pname;
+    rev = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-Ruzi+H8MmIgv23pzFXZlvkk3HtbDzQ9LTTVzmeGWrSI==";
+  };
+
+  buildInputs = [ boost ];
+  propagatedBuildInputs = with epnix.support; [
+    asyn
+    autoparamDriver
+  ];
+
+  # TODO: correct?
+  varname = "ADS";
```

To prevent collision with the other ads driver

```suggestion
  varname = "ADS_DRIVER";
```